DEVELOPER_GUIDE.md

```diff
@@ -44,7 +44,10 @@ To develop on OpenSearch Dashboards, you'll need:
 
 - A [GitHub account](https://docs.github.com/en/get-started/onboarding/getting-started-with-your-github-account)
 - [`git`](https://git-scm.com/) for version control
-- [`Node.js`](https://nodejs.org/), [`npm`](https://www.npmjs.com/), and [`Yarn`](https://yarnpkg.com/) for building and running the project
+- [`Node.js`](https://nodejs.org/), [`npm`](https://www.npmjs.com/), and [`Yarn`](https://yarnpkg.com/) for building and running the project. Later on this document we explain how to install the following versions:
+  - node 18.16.0
+  - npm 9.5.1
+  - yarn 1.22.19
```

Comment:
Including specific version numbers in documentation can introduce a potential anti-pattern, as it necessitates manual upkeep whenever changes occur. Ideally, these version requirements should be automatically validated and ensured by repository tooling.

Reply:
Thanks for the clarification @ruanyl.